data/ocr/regex_brands.txt

```diff
@@ -23,13 +28,16 @@ Carrefour Baby||carrefour [bg]aby
 Carrefour Bio
 Carrefour Discount
 Chabrior
+Coles
```

Comment:
Coleslow ? @raphael0202

Reply:
@teolemon it seems that Coles is indeed a brand: https://au.openfoodfacts.org/brand/coles